src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -54,6 +54,7 @@
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_ClosedFileDiagnostics}" />
                     <CheckBox x:Name="RenameTrackingPreview"
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_RenameTrackingPreview}" />
+                    <Label x:Name="FullSolutionAnalysisMovedToErrorListLabel" Content="Note: Option &quot;Enable full solution analysis&quot; has been moved to the Error List." Visibility="Collapsed"/>
```

Comment:
Should we give a more descriptive message here about the positioning or kind of the button (toggle button)?